MapboxCoreNavigation/Feedback.swift

```diff
@@ -26,22 +26,85 @@ public enum FeedbackType: CustomStringConvertible {
 
     /// Indicates a problem with positioning the user
     case positioning(subtype: PositioningSubtype?)
-    
+
+    /*
+      case streetPermanentlyBlockedOff
+      case roadMissingFromMap
+      case other
+     */
     public var description: String {
         switch self {
         case .general:
             return "general"
-        case .incorrectVisual(_):
+        case .incorrectVisual(subtype: .none):
             return "incorrect_visual"
-        case .confusingAudio(_):
+        case .incorrectVisual(subtype: .turnIconIncorrect):
+            return "turn_icon_incorrect"
```

Comment:
This replaces the type with the subtype in the event. Let’s verify that that’s what the server expects and what the Android navigation SDK is sending. At a glance, I’m unsure how this code would be able to send multiple subtypes for the same event if the user selects multiple rows in the feedback detail view controller – would it be multiple events?

Reply:
As I understand it, the way it sends multiple events is that for each `item` selected in the UI, it creates a new FeedbackEvent (by getting a new UUID for one), then updates that latest event with the actual event details.

Reply:
Based on a cursory reading of the events schema, the SDK needs to send a single feedback event that contains both the type and any subtypes (in the `feedbackSubType` property).